UdpSockets.m

```diff
@@ -82,6 +82,20 @@ @implementation UdpSockets
     if (callback) callback(@[]);
 }
 
+RCT_EXPORT_METHOD(setBroadcast:(NSString*)cId
+                  flag:(BOOL)flag
+                  callback:(RCTResponseSenderBlock)callback) {
+    UdpSocketClient* client = [self findClient:cId callback:callback];
+    if (!client) return;
+
+    NSError *error = nil;
+    if (![client setBroadcast:flag error:&error])
+    {
+        callback(@[error]);
```

Comment:
oops, my bad, got my callbacks confused. No, the way you're emitting the error looks good. I'm still getting that red box error, are you?